From: Michael Biebl <biebl@debian.org>
Date: Mon, 9 Feb 2015 01:12:16 +0100
Subject: build: install nm-settings-ifcfg-rh.5 man page conditionally

Only install nm-settings-ifcfg-rh.5 man page if the ifcfg-rh
configuration plugin has been enabled.
---
 man/Makefile.am | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/man/Makefile.am b/man/Makefile.am
index 4577cb4..5ed5ad5 100644
--- a/man/Makefile.am
+++ b/man/Makefile.am
@@ -67,9 +67,15 @@ docbook_generated_man_pages =		\
 	nmcli-examples.5
 
 docbook_autogenerated_man_pages =	\
-	nm-settings.5			\
-	nm-settings-keyfile.5		\
+	nm-settings.5
+
+docbook_autogenerated_man_pages =	\
+	nm-settings-keyfile.5
+
+if CONFIG_PLUGIN_IFCFG_RH
+docbook_autogenerated_man_pages +=	\
 	nm-settings-ifcfg-rh.5
+endif
 
 EXTRA_DIST +=					\
 	nm-settings.xml				\
